flink-optimizer/src/test/java/org/apache/flink/optimizer/custompartition/CustomPartitioningGlobalOptimizationTest.java

```diff
@@ -57,8 +58,8 @@ public void testJoinReduceCombination() {
 				.withPartitioner(partitioner);
 
 			joined.groupBy(1).withPartitioner(partitioner)
-				.reduceGroup(new IdentityGroupReducerCombinable<Tuple3<Long,Long,Long>>())
-				.print();
+				.reduceGroup(new IdentityGroupReducer<Tuple3<Long,Long,Long>>())
```

Comment:
The semantics of the tests should not be changed. This needs to be changed back to the original `IdentityGroupReducerCombinable`